Plugins/SafeDIGenerateDependencyTree/SafeDIGenerateDependencyTree.swift

```diff
@@ -26,8 +27,20 @@ struct SafeDIGenerateDependencyTree: BuildToolPlugin {
                     .map(\.path)
             }
 
-        let arguments = (targetSwiftFiles + dependenciesSourceFiles).map(\.string)
-        + ["--dependency-tree-output", outputSwiftFile.string]
+        let inputSwiftFilesFilePath = context.pluginWorkDirectory.appending(subpath: "InputSwiftFiles.txt").string
+        try Data(
+            (targetSwiftFiles + dependenciesSourceFiles)
+                .map(\.string)
+                .joined(separator: "\n")
+                .utf8
+        )
+        .write(toPath: inputSwiftFilesFilePath)
+        let arguments = [
+            "--swift-file-paths-file-path",
```

Comment:
I don't love this argument name. If you've got better ideas, I am all ears.

Reply:
- `--swift-sources-file-path`
- `--swift-source-files-file-path`

?

Reply:
ooooh `swiftSources` is good.